src/transforms/reduce/mod.rs

```diff
@@ -338,12 +338,13 @@ impl TaskTransform<Event> for Reduce {
 #[cfg(test)]
 mod test {
     use serde_json::json;
+    use tokio::sync::mpsc;
+    use tokio_stream::wrappers::ReceiverStream;
 
     use super::*;
-    use crate::{
-        config::TransformConfig,
-        event::{LogEvent, Value},
-    };
+    use crate::event::{LogEvent, Value};
+    use crate::test_util::components::assert_transform_compliance;
+    use crate::transforms::test::create_topology;
```

Comment:
Are we preferring the non-nested use statements now?

Reply:
We don't, but I believe the lint that fixes this is only available on nightly, so we have to run it periodically to bring the code-base back to the preferred stylistic choices.